docs/pages/guides/localization.mdx

```diff
@@ -0,0 +1,385 @@
+---
+title: Localization
+---
+
+import { ConfigReactNative } from '~/components/plugins/ConfigSection';
+import { SnackInline } from '~/ui/components/Snippet';
+import { Terminal } from '~/ui/components/Snippet';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { Collapsible } from '~/ui/components/Collapsible';
+
+If you want your app to be easy to use for users that speak different languages or come from different cultures, you should localize it.
+
+Localizing an app makes it adapt to the locale of the user's device. The app will show translations and currencies that the user knows and understands. Numbers, lists and more will format in a way that the user is used to.
+
+## Getting the user's language
+Use the [`expo-localization`](../versions/latest/sdk/localization) module to get the user's current language.
+
+<Terminal cmd={['$ npx expo install expo-localization']} />
+
+```ts
+import { getLocales } from "expo-localization";
+
+const deviceLanguage = getLocales()[0].languageCode;
+```
+
+Expo localization returns the current locale based on the system settings of the device.
+
+On new iOS and Android versions, app language can be set per app, so you usually don't need to build a custom UI to allow users to change the current locale inside of your app. 
```

Comment:
```suggestion
On new Android and iOS versions, app language can be set per app, so you usually don't need to build a custom UI to allow users to change the current locale inside of your app. 
```

[Referecing Android and iOS as per our writing style guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web).